packages/devextreme/js/__internal/ui/chat/chat_message_box.ts

```diff
@@ -59,6 +60,11 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
     this._renderButton();
   }
 
+  _isValuableTextEntered(): boolean {
```

Comment:
```suggestion
  _isValuableTextEntered(): boolean {
    const { text } = this._textArea.option();

    return !!text?.trim();
  }
```

Reply:
@marker-dao  I see it's related to a minor code style only. Don't u mind if I skip it to not rerun tests?

Reply:
Okay

Reply:
[commit](https://github.com/DevExpress/DevExtreme/pull/28003/commits/48ff5cfb1ff572038bda2c381a5085ab30228763)

p.s finally I update it because needed to commit another changes